docs/components/inputMapper.md

```diff
@@ -0,0 +1,216 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.inputMapper`
+
+<!-- TODO: Add links to the messageListener component's documentation -->
+
+This component extends the [`inputMapper.base`](inputMapper.base.md) grade to provide **inter-web page navigation**
+features. It should be used with the `messageListener` component as the navigation-producing invokers rely on background
+scripts to work correctly.
+
+## Using this grade
+
+The component can be used by creating its instance. However, you might face issues as multiple tabs will read gamepad
+inputs at the same time. To avoid those issues, you should use `gamepad.visibilityChangeTracker` in combination with
+`gamepad.inputMapperManager`.
+
+``` javascript
+gamepad.visibilityChangeTracker(gamepad.inputMapperManager);
+```
+
+`gamepad.visibilityChangeTracker` determines whether a browser tab is **visible** or **hidden**. It then passes the
+state of the tab as an argument to the `gamepad.inputMapperManager`, which creates and destroys the instance of
+`gamepad.inputMapper` accordingly.
+
+## Component Options
+
+This component supports the same [configuration options](inputMapper.base.md#component-options) provided by the
+[`inputMapper.base`](inputMapper.base.md) component. You can provide custom configuration options to the component as
+shown in the following example.
+
+``` javascript
+gamepad.visibilityChangeTracker(gamepad.inputMapperManager, {
+    cutoffValue: 0.50,
+    scrollInputMultiplier: 60,
+    model: {
+        map: {
+            buttons: {
+                "0": {
+                    currentAction: "scrollDown",
+                    speedFactor: 2.2
+                }
+            }
+        }
+    }
+});
+```
+
+## Non-navigation Invokers
+
+### `{inputMapper}.restoreFocus()`
+
+- Returns: Nothing.
+
+Restores the focus on the last focused element after the state of browser tab is changed from **hidden** to **visible**
+or after the web page is reloaded due to refreshing or history navigation.
+
+### `{inputMapper}.updateControls()`
+
+- Returns: Nothing.
+
+Updates the gamepad configuration saved by the user.
```

Comment:
I'm not sure I understand the meaning here.  You should probably expand this to cover both what is happening technically (model variables updated, components recreated, whatever) and practically (i.e. what effect if any is noticeable as a result of triggering this invoker).